tests/ConstraintViolationDetectorTest.php

```diff
@@ -15,9 +15,9 @@ class ConstraintViolationDetectorTest extends TestCase
     /** @var ConstraintViolationDetector */
     private $detector;
 
-    public function __construct($name = null, array $data = [], $dataName = '')
```

Comment:
`__construct` is `final` and it takes a single argument.
I moved the body into `setUp()`